crates/proto/src/xfer/dns_exchange.rs

```diff
@@ -193,7 +196,7 @@ where
                     let (dns_request, serial_response): (DnsRequest, _) = dns_request.into_parts();
 
                     // Try to forward the `DnsResponseStream` to the requesting task. If we fail,
-                    // it must be because the requesting task has gone away / is no longer
+                    // it must be because the requesting task has gone away / is no longerwarn!
```

Comment:
:eyes: I think a stray `warn!` may have snuck in to this comment.

Reply:
agh, thanks --- good catch
```suggestion
                    // it must be because the requesting task has gone away / is no longer
```